src/hooks/useScrollPosition.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react'
+
+/** Returns the vertical scroll position. */
+const useScrollPosition = (): number => {
```

Comment:
We currently have a ministore that tracks scrollTop and throttles updates to 60fps: https://github.com/cybersemics/em/blob/main/src/stores/scrollTop.ts

I'm not sure if the throttling will add lag for us. But it certainly would be cleaner and more efficient to use the scrollTop ministore instead of a separate hook if possible. Can you try the ministore and see if there is a performance issue?

```js
const scrollTop = scrollTopStore.useState()
```

Reply:
Ah, my bad. I didn't notice the `scrollTopStore`.

Refactored: 5c38140d2a2e3dbe314612bfb23fd87fe8474dcb